CHANGELOG.md

```diff
@@ -5,6 +5,84 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.5.0] - 2020-04-09
+### Added
+* Overview document for [feature highlights in v0.5.0](https://github.com/Project-MONAI/MONAI/blob/master/docs/source/highlights.md)
+* Invertible spatial transforms
+  * `InvertibleTransform` base APIs
+  * Batch inverse and decollating APIs
+  * Inverse of `Compose`
+  * Test-time augmentation as an application
+* Initial support of learning-based image registration:
+  * Bending energy, LNCC, and global mutual information loss
+  * Fully convolutional architectures
+  * Dense displacement field, dense velocity field computation
+  * Warping with high-order interpolation with C++/CUDA implementations
+* Deepgrow modules for interactive segmentation:
+  * Workflows with simulations of clicks
+  * Distance-based transforms for guidance signals
+* Digital pathology support:
+  * Efficient whole slide imaging IO and sampling with Nvidia cuCIM and SmartCache
+  * FROC measurements for lesion
+  * Probabilistic post-processing for lesion detection
+  * TorchVision classification model adaptor for fully convolutional analysis
+* 12 new transforms, grid patch dataset, `ThreadDataLoader`
+* 4 iteration events for the engine for finer control of workflows
+* New C++/CUDA extensions:
+  * Conditional random field
+  * Fast bilateral filtering using the permutohedral lattice
+* Metrics summary reporting and saving APIs
+* DiceCELoss, DiceFocalLoss, a multi-scale wrapper for segmentation loss computation
+* Data loading utilities：
+  * `decollate_batch`
+  * `PadListDataCollate` with inverse support
+* Support of slicing syntax for `Dataset`
+* Initial Torchscript support for the loss modules
+* Learning rate finder
+* Allow for missing keys in the dictionary-based transforms
+* Various summary and plotting utilities for Jupyter notebooks
+* Contributor Covenant Code of Conduct
+* Major CI/CD enhancements covering the tutorial repository
+* Fully compatible with PyTorch 1.8
+* Initial nightly CI/CD pipelines using Nvidia Blossom Infrastructure
```

Comment:
Added support to load checkpoint with mismatch names and shapes for transfer learning.